test/autorest/stringgroup/string_test.go

```diff
@@ -120,7 +120,7 @@ func TestStringGetWhitespace(t *testing.T) {
 
 func TestStringPutEmpty(t *testing.T) {
 	client := getStringClient(t)
-	result, err := client.PutEmpty(context.Background(), "")
+	result, err := client.PutEmpty(context.Background())
```

Comment:
These changes are due to a change in the swagger not a change in our codegen, right?

Reply:
Correct, it's the change that was discussed in Tuesday's scrum.